Server/ConcordiaCurriculumManager/Controllers/GroupController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using ConcordiaCurriculumManager.DTO;
+using ConcordiaCurriculumManager.Models.Users;
+using ConcordiaCurriculumManager.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ConcordiaCurriculumManager.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+
+public class GroupController : Controller
+{
+    private readonly IGroupService _groupService;
+
+    public GroupController(IGroupService groupService)
+    {
+        _groupService = groupService;
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> CreateGroup([FromBody] GroupCreateDTO groupCreateDTO)
```

Comment:
For this method and the GetGroupById, you return the group object directly. Rather than returning the object directly, can you use Automapper to extract the important attributes to a GroupDTO object, then return the groupDTO? You just have to register the DTO in the AutoMapperProfile class, then there's examples of this mapping in other controllers (it should only take a few lines). In this case you'll map all the attributes over to GroupDTO (Id, Name, and Members), but it's still good practice to return the DTO rather the domain obejct directly as we go forward.